docs/source/web.rst

```diff
@@ -38,6 +38,50 @@ tasks.  For the most recent 20000 tasks visit
 http://my-scheduler-address:8787/tasks , although beware that this page is not
 updated in real time.
 
+Connecting to Web UI
+--------------------
+
+Default
+~~~~~~~
+
+By default, ``dask-scheduler`` will print out 
+
+.. code::
+
+   INFO -  Bokeh UI at:  http://10.129.39.91:8787/status
+   ...
+   INFO - Starting Bokeh server on port 8787 with applications at paths ['/status', '/tasks']
+   
+Try going to that address. In the majority of cases, this should work.
+
+Port forwarding
+~~~~~~~~~~~~~~~
+
+Running distributed on a remote machine can cause issues with viewing the web
+UI -- this depends on the remote machines network configuration.
+
+One way to get around this is via SSH port forwarding. A typical use case looks
+like
+
+.. code:: bash
+
+   local$ ssh -L 8000:localhost:8787 user@remote
+   remote$ dask-scheduler  # now, the web UI is visible at localhost:8000
+   remote$ # continue to set up dask if needed -- add workers, etc
+
+It is then possible to go to ``localhost:8000`` and see Dask Web UI.
+
+This can be used in conjuction with port forwarding on the Jupyter notebook
+port.
```

Comment:
Hrm, perhaps generalize to say something like.  "This same approach is not specific to dask.distributed, but can be used by any service that operates over a network, such as Jupyter notebooks."

Alternatively, perhaps you could write a tiny external blogpost about how many python services like Jupyter and Dask are now providing web interfaces and how you can access them through ssh tunneling.  Then we could link to that page from these docs.

There are enough varied users of Dask that operate in such different ways that I want to make sure we don't get pulled in the game of documenting them all.  I'm afraid that things will start to become confusing as novice users have to figure out what documentation is essential and what documentation was particular to a specific way of using the system.